app/assets/javascripts/resources.js

```diff
@@ -29,20 +29,92 @@ function scrollToTop(event) {
   $('html, body').animate({scrollTop: 0}, 'slow');
 }
 
-function incrementLike(event, target) {
+//getCookie function taken from http://www.w3schools.com/js/js_cookies.asp
+function getCookie(cname) {
+    var name = cname + "=";
+    var ca = document.cookie.split(';');
+    for(var i = 0; i <ca.length; i++) {
+        var c = ca[i];
+        while (c.charAt(0)==' ') {
+            c = c.substring(1);
+        }
+        if (c.indexOf(name) == 0) {
+            return c.substring(name.length,c.length);
+        }
+    }
+    return "";
+}
+
+//setCookie function taken from http://www.w3schools.com/js/js_cookies.asp
+function setCookie(cname, cvalue, exdays) {
+    var d = new Date();
+    d.setTime(d.getTime() + (exdays*24*60*60*1000));
+    var expires = "expires="+d.toUTCString();
+    document.cookie = cname + "=" + cvalue + "; " + expires;
+}
+
+function createNewCookie(target, resourceID){
+  document.cookie = "_shescoding_likes = {}";
+  newValue = JSON.stringify({[resourceID]: true});
+  setCookie("_shescoding_likes", newValue, 365);
+  incrementLikeinDb(target);
+}
+
+function processLike(event, target){
   event.preventDefault();
-  var form = $(target).parents('form');
+  resourceID = target.parentNode.parentNode.action.split('/').slice(-2)[0]
```

Comment:
This needs to be slice(-3)[0]. My mistake for suggesting slice(-2)[0] in an earlier review. I think at that point that did work, because resources without likes did end on '.../<resource-id>/like. It would not have worked when trying to process likes of resources with previous likes. 
Either way, as of the current status, each URL ends on a number behind /like, so slice(-3) should always give the correct result.

Reply:
Ok. Sounds good.